memos.rb

```diff
@@ -2,23 +2,40 @@
 
 require 'sinatra'
 require 'sinatra/reloader'
-require 'json'
+require 'pg'
 require 'cgi'
 
-def save_file
-  File.open('public/json/data.json', 'w') { |file| JSON.dump(@memos, file) }
-end
+# データベースとやり取りをするクラス
+class Memo
+  def initialize
+    @connect = PG::Connection.new(host: 'localhost', user: 'postgres', dbname: 'memo')
+  end
 
-def load_file
-  File.open('public/json/data.json') { |file| @memos = JSON.parse(file.read) }
-end
+  def all
+    @connect.exec('SELECT * FROM memos ORDER BY id ASC')
+  end
+
+  def detail(id)
+    @connect.exec('SELECT * FROM memos WHERE id = $1 LIMIT 1', id)
+  end
 
-def generate_memos(id)
-  @memos[id] = { 'title': params['title'], 'body': params['body'] }
+  def save(title, content)
+    @connect.exec('INSERT INTO memos(title, content) VALUES ($1, $2)', [title, content])
+  end
+
+  def update(title, content, id)
```

Comment:
`id: xxx` の `title` と `content` を `update` するメソッドだと思うので、引数の順番も以下のようになっているのが良いかなと思いました。

```suggestion
  def update(id, title, content)
```

Reply:
引数の順番を変えました。[d95895e](https://github.com/tomonariha/memos/pull/2/commits/d95895e767d9b9f6e6d27fd99139ee4b9d5f1275)